app/views/users/show.html.erb

```diff
@@ -1,7 +1,39 @@
 <h1> <%= @user.username.capitalize %>'s Profile </h1>
 <ul>
   <li>Country of origin: <%= @user.country_of_origin %></li>
-  <li>Gender: <%= @user.gender  %></li>
+  <% unless @user.gender == "Not specified" %>
+    <li>Gender: <%= @user.gender  %></li>
+  <% end %>
   <li>About: <%= @user.about %></li>
-
 </ul>
+
+<h2>Reviews</h2>
+<% @user.reviews.each do |review| %>
+<div>
+  <h4>Review for: <%= review.city.name %>, <%= review.city.country.name %></h4>
+  <li>
```

Comment:
No new `<ul>` or `<ol>` found. Based on the w3c spec, a `<li>` must be contained by an ordered or unordered list.